css/layout.css

```diff
@@ -0,0 +1,30 @@
+
+.photoMe {
```

Comment:
@laceylin2010 best to hyphenate or underscore separate your classes/id's - camel casing is usually reserved for JS var's